Employment Policies/Working Remotely.md

```diff
@@ -1,11 +1,91 @@
+
+When we work remotely, it's our responsibility to accept the benefits graciously, and go above and beyond to not let the costs affect our work or the work of our teammates. A remote workday:
+
+* is not shorter than a day at the office
+* is not less productive
+* is not an excuse for missing commitments (be they meetings or deadlines). 
+
+Even when we aren't in the office, our teammates are still counting on us and it is our responsibility to make sure we are exceeding their expectations.
+
+At the same time, it is the responsibility of workers in the office to reasonably accommodate remote workers. This means being willing to schedule meetings in light of timezone differences and to manage expectations around immediate feedback.  
+
+The guidelines in this document (and the handbook at large) give us tools to make remote work easier, but tooling is only a small part of the solution. All employees need to complement these tools with the extra work required to make being out of the office an effective thing for both the individual remote worker and the team of people who rely on them.
+
+## Policies
+
+The majority of policies that support remote work in Clef apply to everyone at Clef, not just remote workers. The primary documents that outline these policies are:
+
+* [Effective Meetings](https://github.com/clef/handbook/blob/master/Operations%20Documents/Effective%20Meetings.md)
+* [Communication and Transparency](https://github.com/clef/handbook/blob/master/Onboarding%20Documents/Communication%20and%20Transparency.md)
+
+### Extended Remote Work
+
+#### Extended remote work 
+
+***An extended remote work period includes anything longer than 2 days or any period of time where we work from a irregular place.***
+
+If you're planning to work remotely for more than two consecutive days, or you're planning to work remotely from a place that's abnormal (i.e. from a partner's home in a different city), you'll need to follow these additional guidelines. Even more than standard remote work, working remotely for an extended period of time is a privilege — it **will** require a large amount of extra work from you to make sure that the team is effective with you working remotely.
+
+Get Approval From your manager
+
+You need your manager's approval before you can work remotely for an extended period of time. This check-in should happen at least two weeks in advance, before you make any travel arrangements, and you should come prepared with a plan for how you will handle all the edge cases involved in extended remote work. In order for an employee to work remotely for an extended period of time, they should have demonstrated in the past that they are effective working remotely and upholding their quality of work. If that's not the case, the manager can and should veto the option and work with the employee to come up with a plan that would get them on track to work remotely in a more effective way.
```

Comment:
"edge cases" feels weird here

"upholding their quality of work" doesn't make sense